tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
+  sourceMaps: "inline",
+  compact: false,
+  whitelist: [
```

Comment:
Would it be OK to put this off until later, to avoid entangling this project and the react-packages project?